jenkins/gce-custom-images/prepare_custom_leonardo_gce_image.sh

```diff
@@ -124,6 +135,57 @@ add-apt-repository \
   ${os_dist_code_name:?} \
   ${os_dist_release_channel:?}"
 
+# Do some set-up in preparation for image hardening
+
+# Install dpkg-dev so we can use dpkg-architecture down below
+apt-get install -y -q dpkg-dev
+
+# Install Python (needed to install Ansible)
```

Comment:
Do we need to install python 3.7 from source or can we just use the python version that comes with stretch?